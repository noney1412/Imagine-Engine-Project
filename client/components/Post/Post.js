export default () => <div>sdsddsdsdsdsd555</div>;
