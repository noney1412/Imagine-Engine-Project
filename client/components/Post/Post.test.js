import Post from './Post'

test('Post component should render the title not as a link', () => {
  const component = shallow(<Post />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})


