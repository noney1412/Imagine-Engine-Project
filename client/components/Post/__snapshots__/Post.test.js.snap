// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post component should render the title not as a link 1`] = `
<div>
  sdsddsdsdsdsd555
</div>
`;
